### PR TITLE
Added toggle to show/hide extra fields in object details view

### PR DESCRIPTION
**Pull request by bilalabbad**
image
image

Why

  Fields marked with display: "extra" in the schema were always shown in the object details view, cluttering the UI
  with rarely needed information. This PR hides extra fields by default and adds a toggle button to reveal them on
  demand.

  Closes INFP-495

  What changed

  - Extra fields hidden by default: ObjectDataDisplay now filters out fields with display: "extra" unless showExtra is
  explicitly enabled
  - Toggle button in details card: ObjectDetailsCard renders an "Extra" eye icon button when the schema has extra
  fields, toggling their visibility
  - Removed GraphqlQueryDetailsCard: Replaced with the generic ObjectDetailsCard, filtering the "query" attribute
  inline at the call site
  - Convert view always shows extra: ObjectConvert passes showExtra to ensure all fields are visible during object
  conversion

  API contract unchanged — no schema or GraphQL changes.

  How to review

  - Start with has-extra-fields.ts and its tests — simple utility
  - Then object-data-display.tsx for the filtering logic
  - Then object-details-card.tsx for the toggle UI
  - graphql-query-details.tsx for the migration away from the deleted card
  - Tests in object-data-display.test.tsx cover all edge cases

  The deletion of graphql-query-details-card.tsx drops the namespace badge, node label, and edit trigger from the
  GraphQL query detail card header — worth confirming this is acceptable.

How to test

  1. Open any object detail page where the schema has fields with display: "extra" (e.g. a device or interface)
  2. Verify extra fields are not visible by default
  3. Click the Extra toggle button (eye icon) in the "Details" card header
  4. Verify the extra fields now appear
  5. Click the toggle again — extra fields should hide
  6. Navigate to an object that has no extra fields — the toggle button should not appear
  7. Open the object convert view — all fields (including extra) should always be visible
  8. Open a GraphQL query detail page — verify the details card displays correctly without the "query" attribut

  Impact & rollout

  - Backward compatibility: No breaking changes
  - Performance: Negligible — one additional .filter() and .some() per render
  - Deployment notes: Safe to deploy

## Summary by CodeRabbit

* **New Features**
  * Added an "Extra" toggle to object details; fields marked "extra" are hidden by default and shown when toggled
  * Visual indicator (eye icon) marks "extra" fields in the details view
  * GraphQL query details now use the unified object details view and omit the raw query field from display

* **Tests**
  * Added tests covering extra-field detection and show/hide behavior in object data display

### CODE REVIEW

File: frontend/app/src/entities/nodes/object/utils/has-extra-fields.test.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    });
+
+    // WHEN
+    const result = hasExtraFields(schema);
+
+    // THEN
+    expect(result).toBe(true);
+  });
+
+  it("should return false when fields have display values other than 'extra'", () => {
```
Comment (pa-lem) [inline frontend/app/src/entities/nodes/object/utils/has-extra-fields.test.ts]: seems to be a duplication of `should return false when no fields have display 'extra''`

**Review by pa-lem — APPROVED** [review]

File: frontend/app/src/entities/nodes/object/utils/has-extra-fields.test.ts (hunk 1 above)
Comment (bilalabbad) [reply frontend/app/src/entities/nodes/object/utils/has-extra-fields.test.ts]: good catch
